dev-resources/less/lomake.less

```diff
@@ -25,13 +25,27 @@
     margin-left: 1em;
     margin-bottom: 1rem;
   }
+  .lomakeryhman-otsikko-tausta {
+    background-color: @harmaa12;
+    padding-top: 12px;
```

Comment:
Näissä voisi käyttää @valistys-x - muuttujia.

Reply:
Joo, tää oli multa puhtaasti lohkon siirto yhdestä filusta toiseen, niin en hokannu näitä muutella.